starter-code/styles/icons.css

```diff
@@ -68,3 +68,9 @@
 .icon-github::before {
 	content: "\e907";
 }
+
+#hamburger {
+	width: 5%;
```

Comment:
When you do use an icon font, you can change the size by simply changing the font-size, as it is a font :)